Tasks/AzureFileCopy/AzureFileCopy.ps1

```diff
@@ -1,62 +1,42 @@
-param (
-    [string][Parameter(Mandatory=$true)]$connectedServiceNameSelector,
-    [string][Parameter(Mandatory=$true)]$sourcePath,    
-    [string][Parameter(Mandatory=$true)]$destination,
-    [string]$connectedServiceName,
-    [string]$connectedServiceNameARM,
-    [string]$storageAccount,
-    [string]$storageAccountRM,
-    [string]$containerName,
-    [string]$blobPrefix,
-    [string]$environmentName,
-    [string]$environmentNameRM,
-    [string]$resourceFilteringMethod,
-    [string]$machineNames,
-    [string]$vmsAdminUserName,
-    [string]$vmsAdminPassword,
-    [string]$targetPath,
-    [string]$additionalArguments,
-    [string]$cleanTargetBeforeCopy,
-    [string]$copyFilesInParallel,
-    [string]$skipCACheck,
-    [string]$enableCopyPrerequisites,
-    [string]$outputStorageContainerSasToken,
-    [string]$outputStorageURI
-)
+[CmdletBinding()]
+param()
 
 Write-Verbose "Starting Azure File Copy Task"
```

Comment:
Instead of entry and leaving message you can use  Trace-VstsEnteringInvocation $MyInvocation and Trace-VstsLeavingInvocation $MyInvocation command.